modules/bbp-electrophysiology/src/main/resources/schemas/bbp/electrophysiology/trace/v0.1.0.json

```diff
@@ -56,10 +56,10 @@
             "maxCount": 1
           },
           {
-            "path" : "schema:unitCode",
+            "path" : "nsg:dataUnit",
```

Comment:
Looks like 'nsg:dataUnit' is a synonym of 'uniText' or 'unitCode'. I would suggest to use the following shape to constraint 'nsg:dataUnit' :

```json
{
   "path" : "schema:unitCode",
   "name" : "Data unit",
    "description" : "....",
    "or":[
       {
            "datatype": "xsd:string"
        },
       {
           "node": "bbptlotsh:UnitCodeOntologyTermShape"
        }
     ],
     "minCount": 1,
     "maxCount": 1
}
```